### ISSUE TITLE
Error message when opening subscriber mystery item.

### ISSUE THREAD

**Issue by khipkin** [HabitRPG/habitica#8171]
General Info
- UUID: 
- Browser: Chrome
- OS: Windows 10
### Description

I just got my subscriber items, but each time I tried to open one of them, I just got the pictured error message and nothing else.  This happened twice, and it was the same popup both times.

![image](https://cloud.githubusercontent.com/assets/5641207/19709451/b784a16e-9af4-11e6-8865-9804818785fa.png)
#### Console Errors

The Habitica tab had been open for hours, so the Facebook errors probably accumulated slowly over time.

![image](https://cloud.githubusercontent.com/assets/5641207/19709481/eaf015d8-9af4-11e6-83ed-013d7c0c167a.png)
![image](https://cloud.githubusercontent.com/assets/5641207/19709492/fe353b3c-9af4-11e6-97b2-b57835e73c50.png)
![image](https://cloud.githubusercontent.com/assets/5641207/19709504/1315910a-9af5-11e6-927d-a7874856c321.png)
![image](https://cloud.githubusercontent.com/assets/5641207/19709524/2785d4a6-9af5-11e6-972f-d52140a5358f.png)
![image](https://cloud.githubusercontent.com/assets/5641207/19709537/3b8f66b0-9af5-11e6-8a2b-b9ef492803a8.png)

**Comment by khipkin** [comment]
I also just realized that I didn't get the Bailey notification, so maybe that's related?

**Comment by khipkin** [comment]
Almost definitely related to @SabreCat's #8169

**Comment by khipkin** [comment]
Pre-refresh: 

I did not receive the items, but the gift box counter is at zero.  I also did not receive the Bailey notification.
![image](https://cloud.githubusercontent.com/assets/5641207/19709664/5ea58c78-9af6-11e6-8235-4bcf2cf7f4a7.png)
![image](https://cloud.githubusercontent.com/assets/5641207/19709676/6df19a14-9af6-11e6-895f-4700eecba7d3.png)

Post-refresh:

The gift box counter went back to 2 and I successfully opened the items.
However, still no Bailey notification.
